README.md

```diff
@@ -1,94 +1,45 @@
 
-This repo serves as a template for a repository that follows the Scrum process. The following information describes how the native features/workflows of Github can be customized to work in a scrum development process.
+## Hagglehaul Live Version
 
-## Issues
+<https://hagglehaul.azurewebsites.net/>
 
-An issue is a unit of tracking work. Issues can be classified into different classes using `labels`. This can be used to classify issues in the scrum process as follows.
+## What is Hagglehaul
 
-### Epic
+Hagglehaul is a web app built to democratize ridesharing and schedule rides in advance within an era of surge pricing and ride deserts. Users make either a Rider or Driver account to post and place bids respectively on prescheduled trips, where Riders select the winner from the bids they've received to decide on who will be driving and how much it will cost.
 
-An [epic](https://dev.to/jorenrui/a-look-into-how-i-manage-my-personal-projects-my-git-github-workflow-1e7h#epic-issue) is an issue with the label `epic`. It represents a large story that can be broken into stories, which can be addressed over multiple sprints. An epic issue references its story issues as a task list in its description. A Github action has been added to automatically check/uncheck the story task items when they get closed/reopened.
+## Hagglehaul Wiki
 
-### Story
+Guide for Hagglehaul users:
+<https://github.com/RuralBrick/hagglehaul/wiki/User-Guide>
 
-A [story](https://www.atlassian.com/agile/project-management/epics-stories-themes) is an issue with the label `story`. It may represents a new feature, or an enhancement to an existing feature. A story issue can be broken into sub tasks, which are added as a task list in the description of the story issue. These sub task items can be checked manually by the developer to indicate completion.
+Installation/Deployment Guide for Devs:
+<https://github.com/RuralBrick/hagglehaul/wiki/Install-and-Deploy-Guide>
 
-### Bug
+## Hagglehaul Documentation
 
-A bug is an issue with the label `bug`. It represents a problem with the existing code that needs to be fixed.
+Documentation Website:
+<https://ruralbrick.github.io/hagglehaul/>
 
-### Question
+## CI/CD Script Invocation
 
-A question is an issue with the label `question`. It represents a question raised by any one and that may get converted into other types of issues.
+Whenever a developer **pushes a commit(s)**, the nunit-tests.yml script will be invoked. This performs tests on the code and uploads a testing artifact documenting the results to the directory "Hagglehaul-testResults".
 
-## Labels
+Whenever a developer **pushes a branch to "main"**, the azure-webapp-publish.yml script will be invoked. This compiles, packages, and deploys the most current version of the app in "main" to Azure, adding the branch's features to Hagglehaul's live version.
```

Comment:
can you add the "force a re-run" steps here as well? 

1. go into actions
2. click on the most recent run
3. click on "Re-run all jobs"